src/math/Math.sol

```diff
@@ -48,7 +47,14 @@ library Math {
             x := or(x, shr(128, x))
             // Take only the highest bit of x
             x := xor(x, shr(1, x))
+        }
+        return x;
+    }
 
+    /// @dev Returns a particular de Bruijn sequence and log2(x) given that x is a power of 2.
+    function _lookupDeBruijn(uint256 x) internal pure returns (uint256 deBruijnSeq, uint256 y) {
+        deBruijnSeq = 0x00818283848586878898a8b8c8d8e8f929395969799a9b9d9e9faaeb6bedeeff;
```

Comment:
Why is it returned?

Reply:
I think it is for testing purpose..

Reply:
We could have it set as a constant & use it in a test, right?

Reply:
Oh yes good point ! 😆

Reply:
Ok, done ! Gas usage is now 658